types/react-table/index.d.ts

```diff
@@ -733,7 +733,7 @@ export interface UseSortByColumnProps<D extends object> {
     isSortedDesc: boolean | undefined;
 }
 
-export type SortByFn<D extends object> = (rowA: Row<D>, rowB: Row<D>, columnId: IdType<D>) => number;
+export type SortByFn<D extends object> = (rowA: Row<D>, rowB: Row<D>, columnId: IdType<D>, desc: boolean) => number;
```

Comment:
I'm curious if the desc parameter shouldn't be optional since in places such as https://github.com/tannerlinsley/react-table/blob/master/src/publicUtils.js#L21 it's not passed in.

Reply:
I would vote for making it optional since it seems to be an edge-case

Reply:
Ok, updated the pull request